vendor/autoload.php

```diff
@@ -2,6 +2,6 @@
 
 // autoload.php @generated by Composer
 
-require_once __DIR__ . '/composer' . '/autoload_real.php';
+require_once __DIR__ . '/composer/autoload_real.php';
```

Comment:
Just checking that you are only doing the following command in composer:
composer dump-autoload -o
(just noting that when you process via composer always consistently bring in some changes like above; not a big deal and will basically just likely keep flip flopping this stuff depending on who runs the composer update command; just want to make sure you aren't using any custom composer settings or anything like that)